src/components/base-address/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@bcrs-shared-components/base-address",
+  "version": "2.0.0",
```

Comment:
This should be 0.0.0. Lerna will update it.

I'm guessing you think v2 is appropriate as this component is newly hosted (this shared repo instead of common components project). I think that's resonable; when Lerna prompts me then I can select v2.0.0. Other reviewers -- thoughts on this?

Reply:
I guess making it 0.0.0 is the way to go since that's how we usually create new components in this repo? Lerna would do it's wizardry and updates it.

Reply:
Yes, I had it as v2 since it was brought over from common components, but I am fine with changing this to 0.0.0